client/src/components/menus/MainMenu/MainMenu.ts

```diff
@@ -66,7 +65,7 @@ export default class MainMenu extends Vue {
       }),
       () => {
         // TODO: consider if we should just insert current version here
-        window.open('https://csgo-stratbook.s3.eu-central-1.amazonaws.com/Stratbook+Setup+1.8.4.exe');
+        window.open('https://csgo-stratbook.s3.eu-central-1.amazonaws.com/Stratbook+Setup+1.8.3.exe');
```

Comment:
Is there a reason behind this change?

Reply:
1.8.4 doesn't exist yet on S3, only have my macbook with me and can't build NSIS on it :P